all-clouds/index.rst

```diff
@@ -30,10 +30,25 @@ For further details, refer to the cloud-specific documentation:
    .. grid-item-card:: :doc:`Ubuntu on IBM <ibm:index>` 
    .. grid-item-card:: :doc:`Ubuntu on Oracle <oracle:index>`
 
+-----------------------------------------------------------------
 
 
+Other Ubuntu images
+-------------------
 
-------------------------------------------------------------------------------------------------
+Apart from the specific public clouds, Canonical also produces a variety of Ubuntu images for use in other platforms such as LXD, MAAS and OCI container registries.
+
+
+..  grid:: 1 1 2 2
+   :padding: 0
+
+   .. grid-item-card:: :doc:`Ubuntu on OCI container registries <oci:index>`   
+   .. grid-item-card:: :doc:`Ubuntu public images <public-images:index>` 
```

Comment:
Capitalization on Public Images to be in line with our use elsewhere.

Reply:
Oh yeah, good catch!